pkg/controllers/sync/version/manager.go

```diff
@@ -289,6 +295,7 @@ func (m *VersionManager) versionQualifiedName(qualifiedName common.QualifiedName
 func (m *VersionManager) writeVersion(obj pkgruntime.Object, qualifiedName common.QualifiedName) error {
 	key := qualifiedName.String()
 	adapterType := m.adapter.TypeName()
+	keyedLogger := m.logger.WithValues("versionAdapter-type-name", adapterType, "version-qualified-name", key)
```

Comment:
Kebab case MUST be used for keys of structured logging fields.

Reply:
done, pls take review.